helm/cluster-cloud-director/templates/_helpers.tpl

```diff
@@ -110,19 +122,24 @@ joinConfiguration:
     kubeletExtraArgs:
       {{- include "kubeletExtraArgs" . | nindent  6}}
       node-labels: "giantswarm.io/node-pool={{ .pool.name }}"
-{{- if $.Values.proxy.enabled }}
 files:
+{{- if $.Values.proxy.enabled }}
 {{- include "containerdProxyConfig" . | nindent 2}}
 {{- end }}
+{{- if $.Values.network.staticRoutes }}
+{{- include "staticRoutes" . | nindent 2}}
+{{- end }}
 preKubeadmCommands:
 {{- if $.Values.proxy.enabled }}
 - systemctl daemon-reload
 - systemctl restart containerd
 {{- end }}
 postKubeadmCommands:
 {{- if $.Values.network.staticRoutes }}
-{{- include "staticRoutes" . }}
-{{- end -}}
+- systemctl daemon-reload
+- systemctl enable static-routes.service
+- systemctl start static-routes.service
```

Comment:
```suggestion
- systemctl enable --now static-routes.service
```

It's weird to do it differently there

Reply:
ah yeah i was lazy and only made the suggestion in one place

Reply:
I missed it was your suggestion.

Reply:
https://github.com/giantswarm/cluster-cloud-director/pull/66/commits/5c1dfc044fc4d8b22e39c3cdd4c098b3c860e069